fedimint-server/src/consensus/mod.rs

```diff
@@ -118,18 +121,14 @@ pub async fn run(
     let (submission_sender, submission_receiver) = async_channel::bounded(TRANSACTION_BUFFER);
     let (shutdown_sender, shutdown_receiver) = watch::channel(None);
 
-    let mut p2p_status_senders = BTreeMap::new();
     let mut ci_status_senders = BTreeMap::new();
-    let mut status_receivers = BTreeMap::new();
+    let mut ci_status_receivers = BTreeMap::new();
 
     for peer in cfg.consensus.broadcast_public_keys.keys().copied() {
-        let (p2p_sender, p2p_receiver) = watch::channel(P2PConnectionStatus::Disconnected);
         let (ci_sender, ci_receiver) = watch::channel(None);
 
-        p2p_status_senders.insert(peer, p2p_sender);
         ci_status_senders.insert(peer, ci_sender);
-
-        status_receivers.insert(peer, (p2p_receiver, ci_receiver));
+        ci_status_receivers.insert(peer, ci_receiver);
```

Comment:
Side note: I only have a hunch what this could be about, `ConsensusEngine` could use docs